tests/source/macros.rs

```diff
@@ -34,6 +34,13 @@ fn main() {
     a,   
     ];
     
+    vec![a; b];
+    vec!(a; b);
+    vec!{a; b};
+
+    vec![a, b; c];
+    vec![a; b, c];
```

Comment:
Could you add some tests where the expression after the `;` is more complex and then gets reformatted?